src/renderer/editor/properties/icon-picker.ts

```diff
@@ -181,6 +181,8 @@ export class IconPicker extends EventEmitter {
           const iconDiv = theme.createDiv(iconName);
           if (iconName === this.selectedIcon) {
             iconDiv.classList.add('selected');
+            iconDiv.setAttribute('data-bs-toggle', 'tooltip');
+            iconDiv.setAttribute('title', iconName);
```

Comment:
It cannot work this way. The attributes are only added to the icon which is selected when the icon picker is opened. These lines should be moved two line up.